cmd/go-filecoin/address_integration_test.go

```diff
@@ -161,41 +163,29 @@ func TestWalletExportImportRoundTrip(t *testing.T) {
 	_, cmdClient, done := builder.BuildAndStartAPI(ctx)
 	defer done()
 
-	dw := cmdClient.RunSuccess(ctx, "address", "ls").ReadStdoutTrimNewlines()
+	lsJSON := cmdClient.RunSuccess(ctx, "address", "ls").ReadStdoutTrimNewlines()
+	var lsResult commands.AddressLsResult
+	err := json.Unmarshal([]byte(lsJSON), &lsResult)
```

Comment:
It's probably worth folding the JSON unmarshal into something like `RunSuccess(ctx, &lsResult, "address", "ls")`. It will be ubiquitous now.

Reply:
I called it `RunMarshaledJSON`. RunSuccess is still convenient when you don't care about the result type.